Kernel/Storage/SD/Registers.h

```diff
@@ -23,6 +23,13 @@ enum class HostVersion : u8 {
     Unknown
 };
 
+enum class ADMAErrorState : u32 {
```

Comment:
Is there as reason this is declared as `u32` instead of a smaller type like `u8`? I realize this is used as a bit-field below, it just seems strange to use a needlessly large type.

Reply:
Same question about all the `u32` usage in the bitfields here...

Reply:
The bit field is a mmio hardware register, and according to the previous pr, the raspberry pi has a quirk where it only can read in 32 bit chunks form those regions, so when setting the bit field types to 32 bit we tell the compiler to use 32 bit loads, and as a precaution i set the enum type to u32 as well to ensure the same thing.
(Not sure if it actually makes a difference though)

Reply:
(looks like something you could put in the code or at least in the commit message… 🤔)